packages/server/lib/socket-base.ts

```diff
@@ -463,6 +463,9 @@ export class SocketBase {
               return resetRenderedHTMLOrigins()
             case 'cross:origin:automation:cookies:received':
               return this.localBus.emit('cross:origin:automation:cookies:received')
+            case 'request:sent:with:credentials':
+              // NOTE: this is currently a no-op until the server logic is implemented
+              return this.localBus.emit('request:sent:with:credentials', args[0])
```

Comment:
```suggestion
            case 'request:sent:with:credentials':
              return this.localBus.emit('request:sent:with:credentials', args[0])
```

This is no longer a no-op, right? It's used in `server-base.ts`?

Reply:
Yeah this needs to be removed. Good catch!

Reply:
fixed in #24060